core/trino-web-ui/src/main/resources/webapp/src/components/QueryHeader.jsx

```diff
@@ -156,6 +156,14 @@ export class QueryHeader extends React.Component {
                                             JSON
                                         </a>
                                         &nbsp;
+                                        <a
+                                            href={'/ui/api/query/' + query.queryId + '/explain'}
+                                            className="btn btn-info navbar-btn"
+                                            target="_blank"
+                                        >
+                                            Explain
```

Comment:
can you add a screenshot of this new feature to the pr

Reply:
Sure

Reply:
Ok... no worries.